lib/Github/Api/Issue/Labels.php

```diff
@@ -100,17 +100,20 @@ public function deleteLabel($username, $repository, $label)
      * @param string $username
      * @param string $repository
      * @param string $label
-     * @param string $newName
-     * @param string $color
+     * @param array  $params
+     *
+     * @throws \Github\Exception\MissingArgumentException
      *
      * @return array
      */
-    public function update($username, $repository, $label, $newName, $color)
+    public function update($username, $repository, $label, array $params)
```

Comment:
I would suggest to keep the `$newName` and `$color` parameter and add an extra optional `$params` parameter. And merge the new name and color into the params array to keep BC. This way we can merge this change right now and move the 2 parameters to the params array.

Reply:
Sounds good to me. I'll tackle these changes asap.